compiler/lib/generate.ml

```diff
@@ -285,12 +285,14 @@ module Ctx = struct
     ; should_export : bool
     ; effect_warning : bool ref
     ; cps_calls : Effects.cps_calls
+    ; sentinal : Var.t option
```

Comment:
You should use a more explicit name outside of `Deadcode_dgraph`.

Reply:
[8e9f0c2](https://github.com/ocsigen/js_of_ocaml/pull/1503/commits/8e9f0c243b5dfbc727284260c76ddba6d34a553b)